README.md

```diff
@@ -12,96 +12,96 @@ Pull requests are welcome. Please keep the list in alphabetical order.
+[Articulate](http://www.articulate.com/)          | E-learning                               | Ruby, C#/.NET, Javascript                      | ✓                                    | US for engineering      | :keyboard: [Jobs](https://life.articulate.com/careers/)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Articulate-Inc-EI_IE702122.11,25.htm)
+[Auth0](https://auth0.com)                        | Identity Management                      | Node, Web                                      | ✓                                    | [7 countries](https://auth0.com/careers/positions?areas=Engineering)                       | :keyboard: [Jobs](https://auth0.com/jobs)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Auth0-EI_IE1162798.11,16.htm)
 [Automattic](https://automattic.com)              | WordPress & more                         | PHP, Java, Web, Mobile, Node                   | ✓                                    | Anywhere                | :keyboard: [Jobs](https://automattic.com/work-with-us/)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Automattic-EI_IE751107.11,21.htm)
-[Axelerant](https://axelerant.com/)               | Web consulting                           | Drupal, Web                                    | ✗                                    | ?                       | :keyboard: [Jobs](https://axelerant.com/careers/)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Axelerant-EI_IE670528.11,20.htm)
-[Balsamiq](https://balsamiq.com/)                 | Wireframing                              | JavaScript, Web                                | ✗                                    | ?                       | :keyboard: [Jobs](https://balsamiq.com/company/jobs/)
-[Bandcamp](https://bandcamp.com)                  | Music publishing platform                | Ruby, Web                                      | ?                                    | ?                       | :keyboard: [Jobs](https://bandcamp.com/jobs)<br>:door: [Glassdoor](https://www.glassdoor.com/Overview/Working-at-Bandcamp-EI_IE1307633.11,19.htm)
+[Axelerant](https://axelerant.com/)               | Web consulting                           | Drupal, Web                                    | ✗                                    | India                   | :keyboard: [Jobs](https://axelerant.com/careers/)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Axelerant-EI_IE670528.11,20.htm)
+[Balsamiq](https://balsamiq.com/)                 | Wireframing                              | JavaScript, Web                                | ✗                                    | Selected countries      | :keyboard: [Jobs](https://balsamiq.com/company/jobs/)
```

Comment:
Can we be more specific? :slightly_smiling_face:

Reply:
👍